app/views/results/edit.html.erb

```diff
@@ -2,6 +2,9 @@
   <%= stylesheet_link_tag('common/SyntaxHighlighter') %>
   <%= stylesheet_link_tag('grader') %>
   <%= stylesheet_link_tag('pdfjs/viewer') %>
+  <% if !@result.released_to_students %>
+    <%= stylesheet_link_tag('context_menu') %>
```

Comment:
I wonder if you can put a `content_for :head` in `marker/_marker_panes.html.erb` instead of here; that file is rendered only when the submission is still editable, so this would remove the redundant checks.